language/Types.md

```diff
@@ -74,7 +74,16 @@ instance showPercentage :: Show Percentage where
 
 ## Functions
 
-Functions in PureScript are like their Javascript counterparts, but always have exactly one argument.
+Functions in PureScript are like their Javascript counterparts, but always have exactly one argument. Function declarations and lambdas (anonymous functions) with multiple arguments are shorthand for curried lambda abstractions. For example, the following are all equivalent:
+
+```purescript
+add x y = x + y
+add = \x y -> x + y
+add = \x -> \y -> x + y
+add = (+)
+```
+
+Despite this overwhelming convention, uncurried JavaScript functions are can be used through the [purescript-functions](https://pursuit.purescript.org/packages/purescript-functions/) library. For more information see [The Foreign Function Interface](http://www.purescript.org/learn/ffi/).
```

Comment:
Yes, of course we do need to explain this somewhere; I just think it's important to make sure this particular part of our documentation is focused, to keep it concise and to make sure that people don't have to wade through stuff they already know to get to what they're interested in. If they're here, they probably know what they're looking for (remember that this part of the repo is a language reference, and that is how language references are meant to be used). We have PureScript by Example for people who don't have FP backgrounds or indeed anyone new to PureScript - currying and the purescript-functions library are both covered there.

I do think it would be best to revert these changes - although I'd like to hear from other maintainers too.